packages/bpk-tokens/tokens/base.android.xml

```diff
@@ -24,11 +24,11 @@
   <property name="DURATION_SM" category="animations">200ms</property>
   <property name="DURATION_BASE" category="animations">400ms</property>
   <color name="AUTOSUGGEST_LIST_BACKGROUND_COLOR" category="autosuggest">#ffffffff</color>
-  <color name="AUTOSUGGEST_LIST_ITEM_ACTIVE_BACKGROUND_COLOR" category="autosuggest">#e6e4ebff</color>
-  <color name="AUTOSUGGEST_LIST_ITEM_HIGHLIGHTED_BACKGROUND_COLOR" category="autosuggest">#f3f2f5ff</color>
+  <color name="AUTOSUGGEST_LIST_ITEM_ACTIVE_BACKGROUND_COLOR" category="autosuggest">#ffe6e4eb</color>
```

Comment:
Per the [changelog](https://github.com/salesforce-ux/theo/blob/master/CHANGELOG.md#500) it seems like it was using a different transform before. Per the [Android docs](https://developer.android.com/guide/topics/resources/more-resources.html#Color) it seems that `#RRGGBBAA` was never supported and I assume in that case the parse would just not parse the alpha. `#AARRGGBB` is a supported format